packages/tic-tac-toe/package.json

```diff
@@ -113,13 +115,15 @@
     "ember-svg-jar": "2.2.3",
     "ember-template-lint": "1.13.2",
     "ember-truth-helpers": "2.1.0",
+    "emberfire": "^3.0.0-rc.6",
     "enzyme": "3.11.0",
     "enzyme-adapter-react-16": "1.15.2",
     "eslint": "6.8.0",
     "eslint-plugin-ember": "7.7.2",
     "eslint-plugin-node": "11.0.0",
     "eslint-plugin-prettier": "3.1.2",
     "etherlime": "2.2.6",
+    "firebase": "7.11.0",
```

Comment:
Double check this is the same as the one in `rps`

Reply:
It is :)